AxonDeepSeg/apply_model.py

```diff
@@ -0,0 +1,32 @@
+from pathlib import Path
+
+# AxonDeepSeg imports
+import AxonDeepSeg.ads_utils as ads
+from AxonDeepSeg.visualization.merge_masks import merge_masks
+from config import axonmyelin_suffix
+
+from ivadomed import inference as imed_inference
+
+def axon_segmentation(
+                    path_acquisitions_folders, 
+                    acquisitions_filenames,
+                    path_model_folder,
+                    overlap_value=[48,48],
+                    acquired_resolution=None,
+                    segmentations_filenames=[str(axonmyelin_suffix)],
+                    ):
+
+
+    path_model=path_model_folder
+    input_filenames = acquisitions_filenames
+    options = {"pixel_size": [acquired_resolution, acquired_resolution], "overlap_2D":overlap_value, "binarize_maxpooling": True}
+
+    nii_lst, target_lst = imed_inference.segment_volume(str(path_model), input_filenames, options=options)
+
+    imed_inference.pred_to_png(nii_lst, target_lst, str(Path(input_filenames[0]).parent / Path(input_filenames[0]).stem))
+    print(Path(path_acquisitions_folders) / (Path(input_filenames[0]).stem + str(axonmyelin_suffix)))
+    merge_masks(
```

Comment:
Do you mean rename the files, then merge? This comment clear to me right now.

Reply:
Sorry for the confusing comment!
I meant that the naming of the resulting files is a bit odd because they don't all follow the same naming convention.
![image](https://user-images.githubusercontent.com/54086142/145481312-bc6fdc2f-8a16-4342-b089-80813fddd5ba.png)



So we should have the same naming scheme for all of them.

Either the ADS convention:
`_seg-axon.png`
`_seg-axonmyelin.png`
`_seg-myelin.png`

Or the ivadomed convention:
`_seg-axon-manual_pred.png`
`_seg-axonmyelin-manual_pred.png`
`_seg-myelin-manual_pred.png`

Reply:
Done!